text-serializer-ansi/build.gradle

```diff
@@ -0,0 +1,11 @@
+dependencies {
+  api project(':adventure-api')
+  api 'org.fusesource.jansi:jansi:1.18'
```

Comment:
this can be `implementation` scope, right? it's not exposed to users of the API

Reply:
I am not completely gradle fluent but yes...